### ISSUE TITLE
this.$noty.button / VueNoty.button is not a function

### ISSUE THREAD

**Issue by nateritter** [renoguyon/vuejs-noty#1]
I'm attempting to add confirmation buttons to the dialog per https://ned.im/noty/#/confirm.  When I try either `this.$noty.button('YES', 'btn', function () { console.debug('foo'); })` or `VueNoty.button('YES', 'btn', function () { console.debug('foo'); })` and attempt to add it to the message's `buttons` property, I get the following error:

`TypeError: this.$noty.button is not a function`

**Comment by nateritter** [comment]
Similarly, `this.$noty.closeAll is not a function` when trying to `this.$noty.closeAll();`

**Comment by stranger26** [comment]
@nateritter I dont think `.button` is a supported feature in this wrapper at the moment.

**Comment by nateritter** [comment]
@stranger26 Yes, I see that. I guess I should have edited this to be more of a request after I found it.  I'd like buttons to be supported. :)

**Comment by ploissken** [comment]
Is this feature ever going to be supported? (:

**Comment by JonathanSchndr** [comment]
my workaround

```
import 'vuejs-noty/dist/vuejs-noty.css'
import Vue from 'vue'
import VueNoty from 'vuejs-noty'
import Noty from 'noty'

export default ({app}, inject) => {
  inject('noty', Vue.use(VueNoty, {
    timeout: 3000,
    progressBar: false,
    layout: 'topRight',
    closeWith: ['click', 'button'],
  }).noty)

  window.Noty = Noty // workaround for buttons
}
```

**Comment by matt6frey** [comment]
Similar to what @FnnDK was saying, in your main.js, you could probably extend/add that function to noty similar to how you would extend a prototype or object in JS. Perhaps something like:

```javascript
/*
 Other Main.js imports
*/
import Noty from 'noty'
import VueNoty from 'vuejs-noty'

VueNoty.button = Noty.button

Vue.use(VueNoty)

// continue with app initialization
```

**Note** I haven't tested this but it seems like something that would be possible. There may be a little more involvement. Might look into this later tonight 👍
